Create docs for ESM1.6 (#1242)
ACCESS-Hive/access-hive.github.io: docs/includes/payu.md

Reviewer: Might be good to keep the Job ID consistent with the output log below. For example:
```
Job ID: 100.gadi-pbs
::
Output Log: ${CONTROL_DIR}/expt.o100
```

Reviewer: Would it worth mentioning "for machine-readable format, please use `payu status --json`"?

Reviewer: This is correct for payu 1.2.x and 1.3.0). 
However, from payu 1.3.1, `payu status` can monitor collate jobs :)

    `number-of-runs` should be an integer > 0.<br>  
<!--end:payu-run-experiment-->

<!--start:payu-optimise-PBS-->
### Minimise the number of PBS jobs

_Payu_ provides `runspersub` to control the maximum number of runs per PBS job submission.

> `runspersub` allows you to maximise the number of years simulated within a _single_ PBS job, hence reducing the overall queueing time of PBS jobs for an experiment. For this, you also need to set `walltime`, the maximum time of every PBS job, accordingly. In contrast, the command-line option `-n` allows _payu_ to resubmit itself to continue the simulation in a _subsequent_ PBS job.

Reviewer 2: ```suggestion
`runspersub` controls how many years are simulated within a _single_ PBS job, reducing queueing time between jobs. You must also set `walltime` to allow sufficient time for all runs to complete. In contrast, the `-n` command-line option allows _payu_ to resubmit the simulation to a _subsequent_ PBS job.
```

<!--end:payu-optimise-PBS-->

<!--start:payu-runspersub-examples-->
Here are some practical examples of setting these options for different cases. All cases assume `runtime` is set for a 1 year simulation and a model that needs between one and two hours of walltime for a single model year:

- **Run 20 years of simulation with resubmission every 5 years**<br>
    To have a _total experiment length_ of 20 years with a 5-year resubmission cycle, set `runspersub` to `5` and `walltime` to `06:00:00`. Then, run the configuration with `-n` set to `20`:
    ```
    payu run -f -n 20
    ```
    This will submit subsequent jobs for the following years: 1 to 5, 6 to 10, 11 to 15, and 16 to 20, which is a total of 4 PBS jobs.

> - **Run 7 years of simulation with resubmission every 3 years**<br>

Reviewer 2: ```suggestion
- **Run 7 years with resubmission every 3 years**<br>
```

>     To have a _total experiment length_ of 7 years with a 3-year resubmission cycle, set `runspersub` to `3` and `walltime` to `04:00:00`. Then, run the configuration with `-n` set to `7`:

Reviewer 2: ```suggestion
   Set `runspersub` to `3` and `walltime` to `04:00:00`, then run:
```

    ```
    payu run -f -n 7
    ```

>     This will submit subsequent jobs for the following years: 1 to 3, 4 to 6, and 7, which is a total of 3 PBS jobs.

Reviewer 2: ```suggestion
    This submits three PBS jobs covering years 1-3, 4 -6, and 7.
```

<!--start:payu-re-run-experiment-->
### Re-run an experiment from scratch

> If you run an experiment for some years and then realise you have made an error and would like to re-run the same years again, you will need to first remove the _archive_ directory created by _payu_ (i.e., move it, rename it, tar it or delete it according to what is appropriate). You can use `payu sweep --hard` to delete your previous experiment data. You can then re-launch the experiment using `payu run -n N` as done previously.

Reviewer 2: ```suggestion
If you need to rerun years after correcting an error, first remove the archive directory created by payu (e.g., move, rename, archive, or delete it as appropriate). You can use `payu sweep --hard` to remove the previous experiment data, then relaunch the experiment with `payu run -n N` as before.
```

    walltime: 1:00:00
    mpi: false
```

> For models set to generate single restart and output files over the whole grid, the collation is disabled like so:

Reviewer 3: ```suggestion
For configurations set to generate single restart and output files over the whole grid, the collation is disabled like so:
```

```yaml
# Collation
collate:

>     enabled: false

Reviewer 3: Do we need to say this? Isn't collation off by default

<!--start:payu-collate-->
#### Collate {: .no-toc }

> The ocean component [MOM](/models/model_components/ocean/#modular-ocean-model-mom) can generate diagnostic and restart outputs in single files covering the whole model grid or tiled files with each tile spanning a portion of the model horizontal grid.

Reviewer 2: ```suggestion
The ocean component [MOM](/models/model_components/ocean/#modular-ocean-model-mom) can generate diagnostic and restart outputs in single files covering the whole model grid, or tiled files with each tile spanning a portion of the model horizontal grid.
```

    walltime: 1:00:00
    mpi: false
```

> For configurations set to generate single restart and output files over the whole grid, the collation is disabled like so:

Reviewer 2: ```suggestion
For configurations set to generate single restart and output files over the whole grid, the collation is disabled as follows:
```
